runtime/doc/lsp.txt

```diff
@@ -1855,6 +1855,18 @@ typehierarchy({kind})                            *vim.lsp.buf.typehierarchy()*
     Parameters: ~
       • {kind}  (`"subtypes"|"supertypes"`)
 
+workspace_diagnostics({opts})            *vim.lsp.buf.workspace_diagnostics()*
+    Request workspace-wide diagnostics.
+
+    Parameters: ~
+      • {opts}  (`table?`) A table with the following fields:
```

Comment:
Is it worth supporting `vim.lsp.get_clients.Filter` here. E.g. all client_ids attached to this buffer, or client_id's with this name?

Reply:
I think that just filtering by `client_id` is sufficient for now. We can always make this more flexible if requested.